includes/core-changes/corefx/5.0/os-platform-attributes-renamed.md

```diff
@@ -0,0 +1,73 @@
+
+#### Reason for change
+
+.NET 5.0 Preview 8 introduced attributes in <xref:System.Runtime.Versioning> to specify supported platforms for APIs. The attributes are consumed by the [Platform compatibility analyzer](../../../../docs/core/compatibility/code-analysis.md#ca1416-platform-compatibility) to produce build warnings when platform-specific APIs are consumed on platforms that don't supported those APIs.
+
+For .NET 5.0 RC1, an additional feature was added to the platform compatibility analyzer for platform exclusion. The feature allows APIs to be marked as entirely unsupported on OS platforms. That feature prompted changes to the attributes, including using more suitable names. The `ObsoletedInOSPlatformAttribute` was removed because it was no longer needed.
+
+#### Version introduced
+
+5.0 RC1
+
+#### Recommended action
+
+When you retarget your project from .NET 5.0 Preview 8 to .NET 5.0 RC1, you might encounter build or run-time errors due to these changes. For example, the renaming of `MinimumOSPlatformAttribute` is likely to produce errors, because the attribute is applied to platform-specific assemblies at build time, and old build artifacts will still reference the old API name.
+
+Example build-time errors:
+
+- **error CS0246: The type or namespace name 'MinimumOSPlatformAttribute' could not be found (are you missing a using directive or an assembly reference?)**
+- **error CS0246: The type or namespace name 'RemovedInOSPlatformAttribute' could not be found (are you missing a using directive or an assembly reference?)**
+- **error CS0246: The type or namespace name 'ObsoletedInOSPlatformAttribute' could not be found (are you missing a using directive or an assembly reference?)**
+
+Example run-time error:
+
+**Unhandled exception. System.TypeLoadException: Could not load type 'System.Runtime.Versioning.MinimumOSPlatformAttribute' from assembly 'System.Runtime, Version=5.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.**
+
+To resolve these errors:
+
+- Update any references of `MinimumOSPlatformAttribute` to <xref:System.Runtime.Versioning.SupportedOSPlatformAttribute>.
+- Update any references of `RemovedInOSPlatformAttribute` to <xref:System.Runtime.Versioning.UnsupportedOSPlatformAttribute>.
+- Remove any references to `ObsoletedInOSPlatformAttribute`.
+- Rebuild your project (or perform clean + build) to delete old build artifacts.
+
+#### Category
+
+Core .NET libraries
+
+#### Affected APIs
+
+- <xref:System.Runtime.Versioning.MinimumOSPlatformAttribute?displayProperty=fullName>
+- <xref:System.Runtime.Versioning.ObsoletedInOSPlatformAttribute?displayProperty=fullName>
+- <xref:System.Runtime.Versioning.RemovedInOSPlatformAttribute?displayProperty=fullName>
+
+<!--
```

Comment:
Any reason to leave the commented out links here?

Reply:
Yes, they're used by the API analyzer.